### ISSUE TITLE
Changing The Loss Function 

### ISSUE THREAD

**Issue by bluwski** [hunglc007/tensorflow-yolov4-tflite#481]
Hey @hunglc007 , 
I was impressed with your work, I'd like to ask... I saw in the _utils.py_ there are multiple losses (iou, giou, and ciou), how to tryout each losses? Like what script in a function should I replace to use a particular loss?

Thanks
